profiler/src/ProfilerEngine/Datadog.Profiler.Native.Linux/JitCodeCache.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "cor.h" // IWYU pragma: keep
+#include "corprof.h"
+
+#include <array>
+#include <atomic>
+#include <cstdint>
+
+inline constexpr mdFieldDef JitCodeCacheMetadataTokenDummy = 0;
+
+class JitCodeCache
```

Comment:
for now, we really have a large structure, though we do not need everything. This is essentially easier to debug with all of this.